test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NameTest {
+	
+	@Test
+	public void nothing() {
```

Comment:
Hi Justin, great try! But... this commit was a little off since it really does nothing XD. Do note to refrain from such commits for your upcoming projects ~